dockerfiles/Dockerfile.archlinux

```diff
@@ -1,23 +1,19 @@
-FROM archlinux/base
+FROM scratch
 MAINTAINER Iterait a.s. <docker@iterait.com>
 
+COPY --from=archlinux/base / /
+# add cleanup hook
+ADD 99-cleanup.hook /usr/share/libalpm/hooks/
+
 # install common packages
 RUN pacman --noconfirm --needed -Syu \
-      git \
-      base-devel \
-      cmake \
-      gtkglext \
-      hdf5 \
-      gtk3 \
-      opencv \
-      qt5-base \
-      python \
-      vim \
-      python-numpy \
-      python-pip \
-      sudo \
-      boost \
-      openssh
+	binutils \
+	fakeroot \
+        git \
```

Comment:
Also please deindent the lines starting from this one to match the indentation of the previous lines.

Reply:
Dammit, vim!